mm2src/coins/eth.rs

```diff
@@ -4505,7 +4540,7 @@ impl EthCoin {
                     }
                 },
                 EthCoinType::Nft { .. } => {
-                    MmError::err(BalanceError::Internal("Nft Protocol is not supported yet!".to_string()))
+                    MmError::err(BalanceError::Internal("Nft protocol is not supported yet!".to_string()))
```

Comment:
Here instead of BalanceError::Internal we could use a dedicated variant BalanceError::ProtocolNotSupported, like in other errors

Reply:
Its better to revise error variants in other pr